data/PTAM_Kretzschmar_2017.R

```diff
@@ -0,0 +1,58 @@
+# Paper: https://www.researchgate.net/publication/318014800_Re-evaluating_the_psychometric_properties_of_MicroFIN_A_multidimensional_measurement_of_complex_problem_solving_or_a_unidimensional_reasoning_test
```

Comment:
@KingArthur0205 we'll need to update this code potentially after dealing with fractional values. i had merged pull request too quickly